_docs-sources/reference/services/intro/make-changes-to-your-infrastructure.md

```diff
@@ -0,0 +1,181 @@
+   }
+   ```
+
+   To 3 NAT Gateways for high availability:
+
+   ```hcl
+   terraform {
+     source = "git@github.com:gruntwork-io/terraform-aws-service-catalog.git//modules/networking/vpc?ref=v0.0.1"
+   }
+
+   inputs = {
+     # ... (other args omitted for readability) ...
+     num_nat_gateways = 3
+   }
+   ```
+
+1. **(Optional) Update the Service Catalog version**: Another type of change you might make is to update the version
+   of the Service Catalog you're using. For example, perhaps you update from version `v0.0.1`:
+
+   ```hcl
+   terraform {
+     source = "git@github.com:gruntwork-io/terraform-aws-service-catalog.git//modules/networking/vpc?ref=v0.0.1"
+   }
+   ```
+
+   To version `v0.0.2`:
+
+   ```hcl
+   terraform {
+     source = "git@github.com:gruntwork-io/terraform-aws-service-catalog.git//modules/networking/vpc?ref=v0.0.2"
+   }
+   ```
+
+   NOTE: Whenever changing version numbers, make sure to read the [release
+   notes](https://github.com/gruntwork-io/terraform-aws-service-catalog/releases), especially if it's a backwards incompatible
+   release (e.g., `v0.0.1` -> `v0.1.0`), to avoid errors and outages (see [maintenance and
+   versioning](/docs/reference/services/intro/#maintenance-and-versioning) for details)!
+
+1. **Deploy**. Once you've made your changes, you can re-run `terragrunt apply` to deploy them:
+
+   ```bash
+   terragrunt apply
+   ```
+
+   Since Terraform [maintains state](https://blog.gruntwork.io/how-to-manage-terraform-state-28f5697e68fa), it will
+   now update your infrastructure rather than deploying something new. As part of running `apply`, Terraform will show
+   you the plan output, which will show you what's about to change before changing it. Make sure to scan the plan
+   output carefully to catch potential issues, such as something important being deleted that shouldn't be (e.g., a
+   database!).
+
+#### Making changes with Terraform Cloud or Terraform Enterprise
+
+_(Documentation coming soon. If you need help with this ASAP, please contact [support@gruntwork.io](mailto:support@gruntwork.io).)_
+
+### Making changes to Packer templates
+
+1. **(Optional) Update variables**: One type of change you might make is to update the variables you set in your vars
+   file. In particular, you may wish to use a newer version of the Service Catalog for the build. For example, perhaps
+   you update from version `v0.0.1`:
+
+   ```json
+   {
+     "service_catalog_ref": "v0.0.1"
+   }
+   ```
+
+   To version `v0.0.2`:
+
+   ```json
+   {
+     "service_catalog_ref": "v0.0.2"
+   }
+   ```
+
+   NOTE: Whenever changing version numbers, make sure to read the [release
+   notes](https://github.com/gruntwork-io/terraform-aws-service-catalog/releases), especially if it's a backwards incompatible
+   release (e.g., `v0.0.1` -> `v0.1.0`), to avoid errors and outages (see [maintenance and
+   versioning](/docs/reference/services/intro/#maintenance-and-versioning) for details)!
```

Comment:
```suggestion
:::note

Whenever changing version numbers, make sure to read the
[release notes](https://github.com/gruntwork-io/terraform-aws-service-catalog/releases), especially if it's a backwards incompatible release (e.g., `v0.0.1` -> `v0.1.0`), to avoid errors and outages (see [maintenance and
versioning](/docs/reference/services/intro/#maintenance-and-versioning) for details)!

:::
```

Reply:
✅